scripts/setup.sh

```diff
@@ -15,13 +15,17 @@
 # limitations under the License.
 
 K8S_VERSION=$1
+KIND_NAME="kind"
+if [ $# -gt 1 ]; then
+    KIND_NAME=$2
+fi
```

Comment:
Shows that you change it to allow we to define if the tests should or not use kind to address #1435.  Am  I right? I do not think that that it should be in the same PR to propose a new marker for webhooks. Could we address it in a specific PR?